### PR TITLE
Implemented endpoint to remove and edit a playlist in a channel

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
```
stderr: fatal: The remote end hung up unexpectedly
error: RPC failed; result=18, HTTP code = 200
```

Github issue?

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@lfdversluis can you review?

**Comment by lfdversluis** [comment]
looks ok

**Comment by devos50** [comment]
@whirm can you review?

File: Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py [hunk 1]
```diff
@@ -117,3 +120,111 @@ def render_PUT(self, request):
...
+        if playlist is None:
+            return BaseChannelsEndpoint.return_404(request, message="this playlist cannot be found")
+
+        channel_community = self.get_community_for_channel_id(channel_info[0])
+        if channel_community is None:
+            return BaseChannelsEndpoint.return_404(request,
+                                                   message="the community for the specific channel cannot be found")
+
+        # Remove all torrents from this playlist
+        sql = "SELECT dispersy_id FROM PlaylistTorrents WHERE playlist_id = ?"
```
Comment (whirm) [inline Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py]: I don't think the DB stuff should be here.

**Comment by devos50** [comment]
@whirm fixed, see my last mixup commit.

**Comment by whirm** [comment]
Looks fine, squash it

**Comment by devos50** [comment]
```
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Utilities\twisted_thread.py", line 55, in _reactor_runner
    reactor.run(installSignalHandlers=False)
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1194, in run
    self.mainLoop()
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1203, in mainLoop
    self.runUntilCurrent()
--- <exception caught here> ---
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 825, in runUntilCurrent
    call.func(*call.args, **call.kw)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Libtorrent\__init__.py", line 27, in <lambda>
    lambda_f = lambda a = args, kwa = kwargs: invoke_func(*a, **kwa)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Libtorrent\__init__.py", line 28, in invoke_func
    download.session.lm.threadpool.add_task(lambda_f, 1)
exceptions.AttributeError: 'NoneType' object has no attribute 'add_task'
```

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@whirm ready for merge

**Pull request merged**